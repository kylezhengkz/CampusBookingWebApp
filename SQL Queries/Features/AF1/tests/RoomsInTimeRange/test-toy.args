{
    "args": [
        "00000000-0000-0000-0000-000000000001",
        "2019-01-17 08:00:05",
        "2020-12-17 08:00:05",
        null
    ],
    "kwargs": {}
}   