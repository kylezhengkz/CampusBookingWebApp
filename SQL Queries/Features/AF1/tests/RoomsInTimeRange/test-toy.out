{'roomID': UUID('00000000-0000-0000-0000-000000000002'), 'roomName': 'MC 4021', 'bookingCount': 1}
{'roomID': UUID('00000000-0000-0000-0000-00000000000a'), 'roomName': 'Versailled Ball Room', 'bookingCount': 1}